examples/2d/rotation.rs

```diff
@@ -0,0 +1,253 @@
+use bevy::{
+    core::FixedTimestep,
+    math::{const_vec2, Vec3Swizzles},
+    prelude::*,
+};
+
+const TIME_STEP: f32 = 1.0 / 60.0;
+const BOUNDS: Vec2 = const_vec2!([1200.0, 640.0]);
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_system_set(
+            SystemSet::new()
+                .with_run_criteria(FixedTimestep::step(TIME_STEP as f64))
+                .with_system(player_movement_system)
+                .with_system(snap_to_player_system)
+                .with_system(rotate_to_player_system),
+        )
+        .add_system(bevy::input::system::exit_on_esc_system)
+        .run();
+}
+
+/// player component
+#[derive(Component)]
+struct Player {
+    /// linear speed in meters per second
+    movement_speed: f32,
+    /// rotation speed in radians per second
+    rotation_speed: f32,
+}
+
+/// snap to player ship behavior
+#[derive(Component)]
+struct SnapToPlayer;
+
+/// rotate to face player ship behavior
+#[derive(Component)]
+struct RotateToPlayer {
+    /// rotation speed in radians per second
+    rotation_speed: f32,
+}
+
+fn setup(
+    mut commands: Commands,
+    mut materials: ResMut<Assets<ColorMaterial>>,
+    asset_server: Res<AssetServer>,
+) {
+    // Add the game's entities to our world
+
+    let ship_handle = asset_server.load("textures/simplespace/ship_C.png");
+    let enemy_a_handle = asset_server.load("textures/simplespace/enemy_A.png");
+    let enemy_b_handle = asset_server.load("textures/simplespace/enemy_B.png");
+
+    // cameras
+    commands.spawn_bundle(OrthographicCameraBundle::new_2d());
```

Comment:
To make some calculations below perfectly clear, I'd suggest to add a comment describing the coordinate system that this camera produces, and in particular answer the questions:
- where is Y pointing (up or down)?
- where is Z pointing (forward into the screen, or backward into the camera)?

Reply:
I've added doc comments for this function and described Bevy's coordinate system there.